driver/controller.go

```diff
@@ -191,6 +194,15 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	if snapshot != nil && vol != nil && volumeReq.SnapshotID != "" && int(volumeReq.SizeGigaBytes) > int(snapshot.SizeGigaBytes) && int(volumeReq.SizeGigaBytes) > 1 {
+		log.Infof("volume %s will be resize from %d to %d", vol.ID, int(snapshot.SizeGigaBytes), int(volumeReq.SizeGigaBytes))
+		_, resp, err := d.storageActions.Resize(ctx, vol.ID, int(volumeReq.SizeGigaBytes), volumeReq.Region)
```

Comment:
Please handle the action: I know resizes finish immediately today, but that's not guaranteed to be the case always.

Reply:
I tried to add it back but I remembered why I had it removed. I get this error in the logs now : 

May  8 15:18:53 |ERRO| method failed method=/csi.v1.Controller/CreateVolume error=rpc error: code = Internal desc = failed waiting on action ID 1782594914 for volume ID a056bdc3-edb3-11ed-867f-0a58ac14d19d to get resized: timed out waiting for the condition

Reply:
I'd think this must have been a one-off because we also poll the action in the `ControllerExpandVolume` implementation: https://github.com/digitalocean/csi-digitalocean/blob/76367b6a622fe27a517c6ff98c201774040497d2/driver/controller.go#L892-L905